### PR TITLE
Add cfoo gem to Dockerfile

### PR DESCRIPTION
**Pull request by nsarno**
### Background

We started using `cfoo` to create CF stack templates.

### Wat dis do

- Switch docker base image to `ruby`
- Install `cfoo` gem in Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
-FROM gliderlabs/alpine:3.3
+FROM ruby:2.3.1
```
Comment (bentheax) [inline Dockerfile]: no possibility of getting what we need from Alpine ruby packages?

Comment (bentheax) [inline Dockerfile]: surely `ruby:2.3.1-slim` at least is an option

**Comment by bentheax** [comment]
:+1: though preference is for installing ruby onto Alpine unless that's onerous.

**Comment by nsarno** [comment]
@bentheax changed to use alpine

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,15 @@
-FROM gliderlabs/alpine:3.3
+FROM alpine:3.4
```
Comment (iamliamnorton) [inline Dockerfile]: Ohh new alpine nice

File: Dockerfile [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 
-RUN apk --no-cache add \
-      bash \
-      python \
-      py-pip
+ENV BUILD_PACKAGES bash curl-dev ruby-dev build-base
+ENV RUBY_PACKAGES ruby ruby-bundler
+ENV PYTHON_PACKAGES python python-dev py-pip
```
Comment (iamliamnorton) [inline Dockerfile]: I like this separation of packages. I'm going to start using this

**Comment by iamliamnorton** [comment]
👍

File: Dockerfile (hunk 3 above)
Comment (nsarno) [inline Dockerfile]: Yeah I know! Found that somewhere, it's sweet as.

**Pull request merged**